Source/Core/DolphinWX/Frame.cpp

```diff
@@ -690,6 +690,64 @@ WXLRESULT CFrame::MSWWindowProc(WXUINT nMsg, WXWPARAM wParam, WXLPARAM lParam)
 }
 #endif
 
+void CFrame::InhibitScreensaver()
+{
+// Inhibit the screensaver. Depending on the operating system this may also
+// disable low-power states and/or screen dimming.
+
+#if defined(HAVE_X11) && HAVE_X11
+  if (SConfig::GetInstance().bDisableScreenSaver)
```

Comment:
this if condition is the same in all 3 ifdefs, except that Windows sets ES_SYSTEM_REQUIRED even if DisableScreenSaver is false (inhibiting _suspend_).

So should we also rectify that behavior across platforms?

Here's the systemd method for inhibiting suspend:

https://www.freedesktop.org/wiki/Software/systemd/inhibit/
Service org.freedesktop.login1
Interface org.freedesktop.login1.Manager
Path /org/freedesktop/login1
Method Inhibit(string what, string who, string why, string mode) (handle fd)
  what=sleep, who=Dolphin, why=Emulation Running mode=Block

When done, close the returned file.